[rospy.client][INFO] 2017-10-13 16:42:49,614: init_node, name[/distance_collector], pid[3835]
[xmlrpc][INFO] 2017-10-13 16:42:49,614: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-10-13 16:42:49,614: Started XML-RPC server [http://wks-51-57:35484/]
[rospy.impl.masterslave][INFO] 2017-10-13 16:42:49,614: _ready: http://wks-51-57:35484/
[rospy.init][INFO] 2017-10-13 16:42:49,616: ROS Slave URI: [http://wks-51-57:35484/]
[xmlrpc][INFO] 2017-10-13 16:42:49,616: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-10-13 16:42:49,616: Registering with master node http://localhost:11311
[rospy.init][INFO] 2017-10-13 16:42:49,716: registered with master
[rospy.rosout][INFO] 2017-10-13 16:42:49,716: initializing /rosout core topic
[rospy.rosout][INFO] 2017-10-13 16:42:49,718: connected to core topic /rosout
[rospy.simtime][INFO] 2017-10-13 16:42:49,719: initializing /clock core topic
[rospy.simtime][INFO] 2017-10-13 16:42:49,722: connected to core topic /clock
[rospy.internal][INFO] 2017-10-13 16:42:49,731: topic[/clock] adding connection to [http://wks-51-57:40553/], count 0
[rospy.internal][INFO] 2017-10-13 16:42:49,900: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-10-13 16:42:50,046: topic[/all_distances] adding connection to [http://wks-51-57:37445/], count 0
[rospy.internal][INFO] 2017-10-13 16:42:50,110: topic[/all_distances] adding connection to [http://wks-51-57:36406/], count 1
[rospy.core][INFO] 2017-10-13 16:42:53,495: signal_shutdown [signal-2]
[rospy.internal][WARNING] 2017-10-13 16:42:53,498: Unknown error initiating TCP/IP socket to wks-51-57:46456 (http://wks-51-57:36406/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rosgraph/network.py", line 363, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2017-10-13 16:42:53,498: topic[/all_distances] removing connection to http://wks-51-57:36406/
[rospy.internal][WARNING] 2017-10-13 16:42:53,507: Unknown error initiating TCP/IP socket to wks-51-57:32859 (http://wks-51-57:37445/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rosgraph/network.py", line 363, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2017-10-13 16:42:53,507: topic[/all_distances] removing connection to http://wks-51-57:37445/
[rospy.internal][INFO] 2017-10-13 16:42:53,513: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-10-13 16:42:53,513: topic[/clock] removing connection to http://wks-51-57:40553/
[rospy.impl.masterslave][INFO] 2017-10-13 16:42:53,513: signal-2
[rospy.core][INFO] 2017-10-13 16:42:53,629: signal_shutdown [atexit]
